tests/5.0/target/test_target_device.c

```diff
@@ -23,6 +23,7 @@ int test_target_device_ancestor() {
     int i, which_device;
```

Comment:
Minor nit:
```
test_target_device.c:23:9: warning: unused variable ‘i’ [-Wunused-variable]
   23 |     int i, which_device;
      |         ^
test_target_device.c: In function ‘test_target_device_device_num’:
test_target_device.c:60:9: warning: unused variable ‘i’ [-Wunused-variable]
   60 |     int i, target_device_num, host_device_num, first_device_num;
      |         ^
```

Reply:
Removed the unused variable i, as suggested.